### ISSUE TITLE
ciao-cli instance show does not show volumes

### ISSUE THREAD

**Issue by markdryan** [01org/ciao#654]
I created a boot from volume instance.  The instance was created and booted correctly.  But when I did a ciao-cli instance show, no information about the volume was listed.  I'd expect to see the volumes attached to an instance reported by this command.

**Comment by markdryan** [comment]
Perhaps this is a duplicate of #387

**Comment by kaccardi** [comment]
when you specify ciao-cli instance list -detail you should see the attached volumes. I would leave the summary view alone.

**Comment by markdryan** [comment]
I tried this on a hotplugged volume and it wasn't reported when passing --detail to instance list.  Also the show command doesn't have a --detail.

**Comment by markdryan** [comment]
Here's what I've tried.

```
# ciao-cli volume add
Created new volume: f888ae99-b5ea-4274-b556-1eee3ddfd571
# ciao-cli volume attach --volume f888ae99-b5ea-4274-b556-1eee3ddfd571 --instance 3bdf019a-c460-4deb-932d-131bc3ed0fd7
Attached volume: f888ae99-b5ea-4274-b556-1eee3ddfd571
# ciao-cli instance list --detail
Instance #1
	UUID: 3bdf019a-c460-4deb-932d-131bc3ed0fd7
	Status: active
	Private IP: 172.16.0.2
	MAC Address: 02:00:ac:10:00:02
	CN UUID: fe4fa7da-0c46-46cf-9205-28c9d675aa5a
	Image UUID: 73a86d7e-93c0-480e-9c41-ab42f69b7799
	Tenant UUID: f452bbc7-5076-44d5-922c-3b9d2ce1503f
	SSH IP: 198.51.100.52
	SSH Port: 33002
# ciao-cli instance show
# ciao-cli instance show --instance 3bdf019a-c460-4deb-932d-131bc3ed0fd7
	UUID: 3bdf019a-c460-4deb-932d-131bc3ed0fd7
	Status: active
	Private IP: 172.16.0.2
	MAC Address: 02:00:ac:10:00:02
	CN UUID: fe4fa7da-0c46-46cf-9205-28c9d675aa5a
	Image UUID: 73a86d7e-93c0-480e-9c41-ab42f69b7799
	Tenant UUID: f452bbc7-5076-44d5-922c-3b9d2ce1503f
	SSH IP: 198.51.100.52
	SSH Port: 33002
# ciao-cli volume list
Volume #1
	Name             []
	Size             [1 GB]
	UUID             [f888ae99-b5ea-4274-b556-1eee3ddfd571]
	Status           [in-use]
	Description      []

# 
```

**Comment by kaccardi** [comment]
Addressed in PR #700
